common/system/tasks/trigger_manager.cpp

```diff
@@ -52,8 +59,39 @@ void triggerManager_setTriggerFireStatus(uint16_t status) {
   }
 }
 
+/**
+ * Checks if a trigger(Pyro or linecutter) is configured and if that trigger is
+ *connected or not
+ *  TODO add linecutter functionality
+ *  updates triggerConnectivityStatus array
+ **/
+void triggerManager_checkExpectedTriggers() {
+  for (int i = 0; i < NUM_PYRO; i++) {
+    triggerConnectivityStatus[i] = (TriggerState::NOT_CONFIGURED);
+  }
+  SensorData_s* sensorData = hm_getSensorData();
+  int port;
+  for (int i = 0; i < MAX_TRIGGER; i++) {
+#if HAS_DEV(PYRO_CONT)
+    if (cli_getConfigs()->triggerConfiguration[i].mode == TRIGGER_TYPE_PYRO) {
+      port = cli_getConfigs()->triggerConfiguration[i].port;
+      if (sensorData->pyroContData[port]) {
+        triggerConnectivityStatus[port] = (TriggerState::CONNECTED_CONFIGURED);
+      } else {
+        triggerConnectivityStatus[port] =
+            (TriggerState::NOT_CONNECTED_CONFIGURED);
+      }
+      // TODO else if(cli_getConfigs()->triggerConfiguration[i].mode ==
```

Comment:
(nit) Can probably remove this, because there's already the TODO at the top of the function.

Reply:
:)